fmriprep/workflows/bold/base.py

```diff
@@ -231,6 +231,9 @@ def init_func_preproc_wf(bold_file, has_fieldmap=False):
     )
 
     # Find associated sbref, if possible
+    entities = (
+        config.execution.bids_filters.get('sbref', {}) if config.execution.bids_filters else {}
```

Comment:
The idea is to reuse the entities from the BOLD image. I think instead we probably want something like:

```Python
overrides = {
    "suffix": "sbref",
    "extension": [".nii", ".nii.gz"],
}
if config.execution.bids_filters:
    overrides.update(config.execution.bids_filters.get('sbref', {}))
entities.update(overrides)
sbref_files = layout.get(return_type="file", **entities)
```